enlighten/Measurement.py

```diff
@@ -714,7 +714,7 @@ def get_metadata(self, field):
         if field == "blank":                     return self.settings.eeprom.serial_number # for Multispec
         if field == "note":                      return self.save_options.note() if self.save_options is not None else ""
         if field == "temperature":               return self.processed_reading.reading.detector_temperature_degC if self.processed_reading.reading is not None else -99
-        if field == "technique":                 return self.technique
```

Comment:
I think probably all of these should have stayed "Technique."  In the UI, we're changing the "Technique" comboBox to a "View" comboBox.  However, we really do still need to keep track of Technique — it's still a valid concept, and it is what Measurement needs to know.

If we don't already have one, SpectrometerApplicationState should have a .technique member which can be an enum from the set { NONE, RAMAN, REFLECTANCE_TRANSMISSION, ABSORBANCE, COLOR, FLUORESCENCE, RELATIVE_IRRADIANCE } (for now).  All of those but "RAMAN" would be selectable from a new "Technique" widget in the ControlPanel.  The Technique widget should only be visible in Non-Raman and Expert Modes; when in Raman mode, the technique should automatically be set to Raman.  And though this slightly complicates things, if in Expert mode, then the comboBox should probably allow Raman to be selected as well.

Sorry for not making this clear as we continue to think-through what this design might involve.